config/actions/retrieve-secrets/action.yml

```diff
@@ -10,21 +10,21 @@ runs:
   using: "composite"
   steps:
     - name: Configure AWS credentials
-      uses: aws-actions/configure-aws-credentials@v1-node16
+      uses: aws-actions/configure-aws-credentials@e1e17a757e536f70e52b5a12b2e8d1d1c60e04ef # pin@v1-node16
```

Comment:
This should be updated to a more recent, non-node16 version

Reply:
Done, updated to latest version.